packages/fluentui/react-northstar-prototypes/tsconfig.json

```diff
@@ -0,0 +1,37 @@
+{
+  "extends": "../../../scripts/typescript/tsconfig.fluentui",
+  "compilerOptions": {
+    "composite": true,
+    "outDir": "dist/dts"
+  },
+  "paths": {
+    "@fluentui/keyboard-key": ["packages/keyboard-key/src/index"]
+  },
+  "include": ["src"],
+  "references": [
```

Comment:
Let's remove `references` that are not required

Reply:
I got weird error when I don't have it. Looks like `yarn build` is trying to build every @fluentui dependency.